tasks/pkg-windows.yml

```diff
@@ -14,11 +14,11 @@
     Set-ExecutionPolicy Bypass -Scope Process -Force
     &$env:temp\fix_6_14.ps1
 
-- include_tasks: win_agent_6_latest.yml
-  when: not datadog_agent5 and (datadog_agent_version | length == 0)
+- include_tasks: win_agent_latest.yml
+  when: datadog_agent_major_version > 5 and (datadog_agent_version | length == 0)
```

Comment:
No need to check `datadog_agent_major_version` here and in the next task since we fail the run line 1 is A5 is set.